pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -316,32 +316,32 @@ var _ = Describe("Generating fluentd config", func() {
 				</match>
 				<filter kubernetes.**>
 					@type kubernetes_metadata
-					kubernetes_url "#{ENV['K8S_HOST_URL']}"
-					cache_size "#{ENV['K8S_METADATA_CACHE_SIZE'] || '1000'}"
-					watch "#{ENV['K8S_METADATA_WATCH'] || 'false'}"
-					use_journal "#{ENV['USE_JOURNAL'] || 'nil'}"
-					ssl_partial_chain "#{ENV['SSL_PARTIAL_CHAIN'] || 'true'}"
+					kubernetes_url 'https://kubernetes.default.svc'
+					cache_size '1000'
+					watch 'false'
+					use_journal 'nil'
+					ssl_partial_chain 'true'
 				</filter>
 
 				<filter kubernetes.journal.**>
 					@type parse_json_field
-					merge_json_log "#{ENV['MERGE_JSON_LOG'] || 'false'}"
-					preserve_json_log "#{ENV['PRESERVE_JSON_LOG'] || 'true'}"
-					json_fields "#{ENV['JSON_FIELDS'] || 'MESSAGE,log'}"
+					merge_json_log 'false'
+					preserve_json_log 'true'
+					json_fields 'log,MESSAGE'
```

Comment:
Is the order of `log` and `MESSAGE` important for the filter plugin?

Reply:
I don't believe so.  we have examples of where this is flipped the other way but it is the same fields